packages/ui/src/components/sign-in/sign-in.tsx

```diff
@@ -735,6 +735,7 @@ export function SignInComponentLoaded() {
                     </Common.GlobalError>
                     <div className='flex flex-col justify-center gap-4'>
                       <PasswordField
+                        validatePassword
```

Comment:
Wasn't too sure if I needed to add this to the `confirmPassword` field below; my gut is telling me no

Unfortunately, I can't test this view due to a [recent regression](https://linear.app/clerk/issue/SDKI-151/signin-forgot-password-no-longer-able-to-see-actions)